modules/osdk-run-deployment.adoc

```diff
@@ -56,19 +59,69 @@ $ make docker-push IMG=<registry>/<user>/<image_name>:<tag>
 The name and tag of the image, for example `IMG=<registry>/<user>/<image_name>:<tag>`, in both the commands can also be set in your Makefile. Modify the `IMG ?= controller:latest` value to set your default image name.
 ====
 
+ifdef::java[]
+. Run the following command to install the CRD to the default namespace:
++
+[source,terminal]
+----
+$ oc apply -f target/kubernetes/memcacheds.cache.example.com-v1.yml
+----
++
+.Example output
+[source,terminal]
+----
+customresourcedefinition.apiextensions.k8s.io/memcacheds.cache.example.com created
+----
+
+. Create a file called `rbac.yaml` as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: memcached-operator-admin
+subjects:
+- kind: ServiceAccount
+  name: memcached-quarkus-operator-operator
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+----
++
+[IMPORTANT]
+====
+The `rbac.yaml` file will be applied at a later step.
+====
+
+endif::[]
+
 . Run the following command to deploy the Operator:
 +
 [source,terminal]
 ----
 $ make deploy IMG=<registry>/<user>/<image_name>:<tag>
```

Comment:
Could you please add the below information as a note?

Note: The ServiceAccount from the `target/kubernetes/kubernetes.yml` file does not contain any namespace that's why it gets created in the `default` namespace. Please update `ClusterRoleBinding` as below.

```
subjects:
  - kind: ServiceAccount
    namespace: default
    name: memcached-operator-operator
```